samples/boards/nrf/ieee802154/802154_rpmsg/prj.conf

```diff
@@ -5,4 +5,4 @@ CONFIG_UART_CONSOLE=n
 CONFIG_LOG=n
 
 CONFIG_NRF_802154_SER_RADIO=y
-CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=2
+CONFIG_NRF_RTC_TIMER_USER_CHAN_COUNT=3
```

Comment:
Why not just remove this entry? `NRF_802154_SER_RADIO` selects `NRF_802154_RADIO_DRIVER`, so `NRF_RTC_TIMER_USER_CHAN_COUNT` will be 3 by default.

Reply:
Ok, I like such a cleanup. Entry removed.
However, in the case of `CONFIG_NRF53_SYNC_RTC=y`, it turned out that the default value is from `soc/arm/nordic_nrf/nrf53/Kconfig.soc`, so I made a change there.